src/App.jsx

```diff
@@ -1,35 +1,20 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import TodoList from './TodoList'
+import AddTodoForm  from './AddTodoForm'
+import 'bootstrap/dist/css/bootstrap.css'
+
 
 function App() {
-  const [count, setCount] = useState(0)
+ 
 
   return (
```

Comment:
Looks like you're missing a heading here. It's in your components which actually isn't a bad idea to encapsulate that within the component it is associated with but for now lets stick to the instructions. Good instinct.